composer.json

```diff
@@ -70,13 +70,11 @@
         "jms/serializer": "<0.13",
         "liip/imagine-bundle": "<1.9",
         "rackspace/php-opencloud": "<1.6",
-        "sonata-project/block-bundle": "<3.17 || >=4.0",
```

Comment:
Should the conflict against <3.17 also be removed?

Reply:
It should not.

Reply:
Why the Seo 3.0 should be needed ? We're on master, can't we using dev-master ?

Reply:
We can use dev-master. SEO have only some little things to do, do why we should wait with major release?

Reply:
Don't know about others bundle than AdminBundle and persistence bundle.

But I thought the major release was waiting the AdminBundle 4 support.
And we should use the major release to restrict visibility of class/property/function.

Reply:
SeoBundle is more like BlockBundle.